content/en/docs/concepts/configuration/secret.md

```diff
@@ -1244,83 +1227,36 @@ these pods.
 
 ## Information security for Secrets
 
-Although ConfigMap and Secret work similarly, Kubernetes applies some additional
-protection for Secret objects.
+Although ConfigMaps and Secrets work similarly, Kubernetes applies some
+additional protections to Secret objects.
 
-Secrets often hold values that span a spectrum of importance, many of which can
-cause escalations within Kubernetes (e.g. service account tokens) and to
-external systems. Even if an individual app can reason about the power of the
-Secrets it expects to interact with, other apps within the same namespace can
-render those assumptions invalid.
+Secrets often hold values that span a spectrum of importance. Many of these
+values can cause escalations within Kubernetes and to external systems. Even if
+an individual app can reason about the power of the Secrets it expects to
+interact with, other apps within the same namespace can render those assumptions invalid.
 
 A Secret is only sent to a node if a Pod on that node requires it.
-For mounting secrets into Pods, the kubelet stores a copy of the data into a `tmpfs`
-so that the confidential data is not written to durable storage.
-Once the Pod that depends on the Secret is deleted, the kubelet deletes its local copy
-of the confidential data from the Secret.
+For mounting secrets into Pods, the kubelet stores a copy of the data into a
+`tmpfs` so that the confidential data is not written to durable storage.
```

Comment:
This can be covered after PR is merged, but it would be good to clarify that "tmpfs" == "in-memory storage" != "stored at rest"

Reply:
Slightly related: if you turn on `NodeSwap` _or_ run the kubelet with the otherwise-unsupported use of a local pagefile, tmpfs data _can_ be stored at rest.

(consider not enabling swap if this behavior isn't what you want; otherwise, encrypt your page files, folks)